### DISCUSSION TITLE
Sorry: Cannot remember the answer??

### DISCUSSION THREAD

**Discussion by ra49brla** [tanujnotes/Olauncher#426] [Q&A] [answered]
I have Olauncher installed on my tablet and also Motorola phone ❤️❤️❤️ so much more peaceful in my equipment ever since. 
And you told me how to change the default left and right swipes, so those are now gallery (left) and camera (right). 
I have a new Nokia phone but cannot remember what you said to do, to change the swipe defaults. 🥺 I have turned them on and off a dozen times (sometimes phones don't see my fingertips). 
Sorry to bother you again, but please tell me again. I will save the answer somewhere safe this time.

**Comment by tanujnotes** [comment, marked as answer]
Long press anywhere on the home screen to open settings. Tap the app names next to Swipe left/right app and select another one from the list. Hope this helps. Let me know if you have any other question.

Thank you for the coffee!

**Reply by ra49brla** [reply]
Thank you so much 🙏 hope it was good coffee .
